mesonbuild/envconfig.py

```diff
@@ -299,6 +301,10 @@ def is_hurd(self) -> bool:
         """
         return self.system == 'gnu'
 
+    def is_irix(self) -> bool:
+        """Machine is IRIX?"""
+        return self.system == 'irix64'
```

Comment:
I know that irix64 is already pretty old, but maybe this should be `return self.system.startswith('irix')` just in case someone tries to use meson on an old iris 32bit system?